HandlePubKeyRevoke/handler.ts

```diff
@@ -164,14 +164,22 @@ export const handleRevoke = (
       trackException(telemetryClient, {
         exception: new Error(error),
         properties: {
+          /* eslint-disable sort-keys */
```

Comment:
Can we sort properties? Is `sort-keys` disabled for some reason?

Reply:
Just because I don't like it  so much :) I'll fix it

Reply:
Done in [21a7159](https://github.com/pagopa/io-functions-lollipop/pull/37/commits/21a71591e53b5c92d31933e725e786c329bf146d)